sanic/app.py

```diff
@@ -735,6 +736,18 @@ async def handle_exception(
             context={"request": request, "exception": exception},
         )
 
+        if isinstance(request.stream, Http) and request.stream.stage in (
+            Stage.RESPONSE,
+            Stage.IDLE,
+        ):
+            error_logger.exception(exception)
+            logger.error(
+                "The error page was not sent to the client for the "
+                "exception raised above because a previous response has "
```

Comment:
The loggers may not be going to the same location, so we should not say "above"

```python
f"exception raised ({exception}) because ...
```

Reply:
fixed, the new one looks like:

```python
logger.error(
    f"The error response won't be sent to the client for the "
     {exception} because a previous response has "
    "already been sent at least partially."
)
```